lib/server.ml

```diff
@@ -266,6 +278,22 @@ let rec input_userauth_request t username service auth_method =
     else
       handle_auth t
 
+let reject_userauth t _userauth =
```

Comment:
The reason for the unused `userauth` argument is that in a .mli we can better ensure proper protocol flow as you can only call it if you've got a `userauth` in your hand - which, if made opaque, you can only get from a `Userauth _` event.